lnp2p/src/bolt/bolt9.rs

```diff
@@ -547,6 +547,13 @@ impl InitFeatures {
         }
         map
     }
+
+    pub fn new() -> InitFeatures {
```

Comment:
This makes `InitFeatures::new` diverged from `InitFeatures::default`, which is a non-obvious. Either `new` should be re-named into `InitFeatures::with_loss_protect` - or `Default` should be removed. I think the first is more desired, since it is not clear why we silently opting into a single feature and not the rest.

Reply:
I'm sorry, this shouldn't have publish in the repository, because is only a test. I will remove it.